### PR TITLE
Load remote urls

### PR DESCRIPTION
**Pull request by floryst**
This adds the ability to load remote datasets via URL parameters.

- [x] Add documentation

### CODE REVIEW

File: src/composables/useAppLoadingNotifications.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        closeOnClick: false,
+      });
+    }
+  };
+
+  const stopLoading = () => {
+    loadingCount--;
+    if (loadingCount === 0 && toastID !== null) {
+      if (error) {
+        toast.dismiss(toastID);
```
Comment (PaulHax) [inline src/composables/useAppLoadingNotifications.ts:39]: Just for my learning: Why did you not throw a bang in `toast.dismiss(toastID!)` and take the runtime error? If `toastID === null` then startLoading would not have been called, which would be a silent bug now?

File: src/composables/useAppLoadingNotifications.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import { TYPE } from 'vue-toastification';
+import { useToast } from '@/src/composables/useToast';
+import { ToastID } from 'vue-toastification/dist/types/src/types';
+import { useMessageStore } from '../store/messages';
+
+export function useAppLoadingNotifications() {
```
Comment (PaulHax) [inline src/composables/useAppLoadingNotifications.ts:6]: Nice abstraction!

**Review by PaulHax — APPROVED** [review]
Works good!

File: src/utils/index.ts [hunk 3]
```diff
@@ -41,6 +41,19 @@ export async function fetchFileWithProgress(
   return new File([bytes], name);
 }
 
+/**
+ * Fetches a file and returns a Blob.
```
[the remark this answers is not in the document]
Comment (floryst) [reply src/utils/index.ts:45]: Whoops, that's supposed to be "returns a File".

File: src/composables/useAppLoadingNotifications.ts (hunk 1 above)
Comment (floryst) [reply src/composables/useAppLoadingNotifications.ts:39]: Just defensive programming. Also, I have another project where the linter complains about non-null assertions, so I likely implicitly followed that here.

**Pull request merged**